### PR TITLE
finished 24Hr ticker endpoint api call

### PR DESCRIPTION
**Pull request by jay10611**

### CODE REVIEW

File: CryptoCore/Controllers/HomeController.cs [hunk 1]
```diff
@@ -35,9 +35,19 @@ public async Task<IActionResult> Index()
...
+            var response = await _binanceClient.GetTwentyFourHourTickerInfo();
+            foreach (var coin in response)
+            {
+                var tempObject = new TickerModel();
+                tempObject.Symbol = coin.symbol;
+                tempObject.PriceChangePercent = coin.priceChangePercent;
+                model.Coins.Add(tempObject);
+
+            }
+            return View(model);
```
Comment (Michch333) [inline CryptoCore/Controllers/HomeController.cs]:
```suggestion
            return View();
```

Comment (Michch333) [reply CryptoCore/Controllers/HomeController.cs]: Keep the controllers clean until we need to send the data by. Less merge conflicts

File: CryptoCore/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
+@model CurrentPriceViewModel
 <div class="text-center">
     <h1 class="display-4">Welcome</h1>
     <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
 </div>
+@foreach(var coin in Model.Coins)
+{
+    <p>@coin.Symbol</p>
+    <p>@coin.Price</p>
+}
```
Comment (Michch333) [inline CryptoCore/Views/Home/Index.cshtml:13]: This is probably fine for now

File: CryptoCore/Views/Home/Privacy.cshtml [hunk 3]
```diff
@@ -2,5 +2,10 @@
...
 }
 <h1>@ViewData["Title"]</h1>
-
+@model TickerViewModel
 <p>Use this page to detail your site's privacy policy.</p>
+@foreach(var coin in Model.Coins)
+{
+    <p>@coin.Symbol</p>
+    <p>@coin.PriceChangePercent</p>
+}
```
Comment (Michch333) [inline CryptoCore/Views/Home/Privacy.cshtml]: Keep it clean :)

```suggestion

```

**Review by Michch333 — CHANGES_REQUESTED** [review]

**Review by jay10611 — COMMENTED** [review]
im about it

**Review by Michch333 — APPROVED** [review]

**Pull request merged**